### PR TITLE
RA-1504

### PR DESCRIPTION
**Pull request by slubwama**
https://issues.openmrs.org/browse/RA-1504
Enhancement to allow use of patient id or uuid when accessing mark patient dead page.

### CODE REVIEW

**Comment by slubwama** [comment]
@dkayiwa here is a PR that breaks down the issues of [RA-134](https://github.com/openmrs/openmrs-module-coreapps/pull/147)
into two as requested.

**Comment by slubwama** [comment]
@dkayiwa The other ticket still has work to do in it so I cant close it

**Comment by dkayiwa** [comment]
Sorry i meant pull request 😊

**Comment by slubwama** [comment]
@dkayiwa Its done. will recreate it when this is merged.

File: omod/src/main/java/org/openmrs/module/coreapps/page/controller/MarkPatientDeadPageController.java [hunk 1]
```diff
@@ -27,21 +27,20 @@
 public class MarkPatientDeadPageController {
     protected final Log log = LogFactory.getLog(this.getClass());
 
-    public void get(@SpringBean PageModel pageModel, @RequestParam(value = "breadcrumbOverride", required = false) String breadcrumbOverride, @SpringBean("patientService") PatientService patientService, @RequestParam("patientId") String patientId) {
+    public void get(@SpringBean PageModel pageModel, @RequestParam(value = "breadcrumbOverride", required = false) String breadcrumbOverride, @SpringBean("patientService") PatientService patientService, @RequestParam("patientId") Patient patient) {
```
Comment (dkayiwa) [inline omod/src/main/java/org/openmrs/module/coreapps/page/controller/MarkPatientDeadPageController.java]: Didn't we agree that the above change is gonna be in a separate pull request?

Comment (slubwama) [reply omod/src/main/java/org/openmrs/module/coreapps/page/controller/MarkPatientDeadPageController.java]: @dkayiwa  this is the separate PR. with only one issue. I don't understand how micro you want this to get.

Comment (dkayiwa) [reply omod/src/main/java/org/openmrs/module/coreapps/page/controller/MarkPatientDeadPageController.java]: Oh i see!
I now get you. 😊
